src/main/java/com/laytonsmith/core/events/drivers/PlayerEvents.java

```diff
@@ -910,6 +910,67 @@ public boolean matches(Map<String, Mixed> prefilter, BindableEvent e) throws Pre
 			return false;
 		}
 
+		@Override
+		public PrefilterBuilder getPrefilters() {
+			return PrefilterBuilder
+					.start("button", PrefilterType.STRING_MATCH, new PrefilterMatcher() {
```

Comment:
This is an example of a prefilter where case is current insensitive.

Reply:
Oh I see. Yeah, that's just because the value is lowercased. Ok, good catch, I'll have to give this some thinks.